.github/workflows/lint.yml

```diff
@@ -0,0 +1,14 @@
+name: lint
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2
```

Comment:
```suggestion
      - uses: pre-commit/action@v2.0.2
```

Reply:
Ok, I guess dependabot will have a bit more work to do.

As reference : https://github.com/pre-commit/action/issues/104#issuecomment-843821467